books/book.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../item'
+require 'date'
+
+class Book < Item
+  attr_accessor :title, :publish_date, :publisher, :cover_state, :label
+
+  def initialize(title, publish_date, publisher, cover_state, label: nil)
+    super(title, publish_date)
+    @title = title
+    @publish_date = publish_date
```

Comment:
since the `title` and `publish_date` are being passed to the parent constructor, these two lines are not necessary it will be a duplicate thing.
It's better to remove them.

Reply:
Ok great....thanks for the feedback.